modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -67,6 +67,20 @@ namespace detail {
         Kind kind;
         bool is_generic;
         IEConfig config;
```

Comment:
> IEConfig config
> std::string device_id;

BTW, Why IEConfig doesn't hold the device information?

/cc @dmatveev

Reply:
`IEConfig` here holds the keys for [`IE::Core::SetConfig()`](https://docs.openvinotoolkit.org/latest/classInferenceEngine_1_1Core.html#a34aa9ac6fb237b634d5bf08b288e88d4) method only, so the device information is kept separately

Reply:
Usually such config keys are device-specific.
Also `.SetConfig()` call has 2 parameters, the second one is a "device".

Reply:
Device is specified separately, in the param structure constructor (as well as path to .xml/.bin)